examples/bsoncxx/builder_list.cpp

```diff
@@ -61,11 +61,12 @@ int main(int, char**) {
     //  }
     //
     // clang-format off
-    doc = {"BSON boolean value", false,
-           "BSON 32-bit signed integer value", -123,
-           "BSON date value", std::chrono::milliseconds(123456789),
-           "BSON Decimal128 value", decimal128{100, 200},
-           "BSON regex value with options", bson_value::value("regex", "imsx" /* opts */)};
+    doc = {{"BSON boolean value", false},
+           {"BSON 32-bit signed integer value", -123}};
+
+    doc += {{"BSON date value", std::chrono::milliseconds(123456789)},
```

Comment:
How do you create a document that holds an array now?  Like `{ "fruits" : [ "apple", "banana" ] }`

Reply:
Acknowledged